src/vs/workbench/services/editor/common/editorResolverService.ts

```diff
@@ -59,6 +59,21 @@ export interface IEditorType {
 configurationRegistry.registerConfiguration(editorAssociationsConfigurationNode);
 //#endregion
 
+//#region defaultBinaryEditor
+export const defaultBinaryEditorSettingId = 'workbench.defaultBinaryEditor';
+
+const defaultBinaryEditorConfigurationNode: IConfigurationNode = {
+	...workbenchConfigurationNodeBase,
+	properties: {
+		'workbench.defaultBinaryEditor': {
```

Comment:
We typically use `workbench.editor.xy` as namespace for editor related settings.

Reply:
Renamed. I was just following the way `editorAssociations` is. Too late to rename that one though